libminifi/src/utils/OsUtils.cpp

```diff
@@ -24,16 +24,18 @@
 #ifndef WIN32_LEAN_AND_MEAN
 #define WIN32_LEAN_AND_MEAN
 #endif
-#include <Windows.h>
 #include <sddl.h>
+#include <Windows.h>
```

Comment:
Judging from the appveyor output, the order of these includes matter. Not sure of the exact reason, but in general windows headers are a mess, so I'm not surprised. :(

See the end of the log here: https://ci.appveyor.com/project/ApacheSoftwareFoundation/nifi-minifi-cpp/builds/33004452

Reply:
Corrected.